proof/refine/ARM_HYP/ArchVSpace_R.thy

```diff
@@ -4,30 +4,17 @@
  * SPDX-License-Identifier: GPL-2.0-only
  *)
 
-(*
-   ARM VSpace refinement
-*)
+(* ARM VSpace refinement *)
```

Comment:
Should this be `ARM_HYP`?

I guess the other option would be for this comment to say something like `VSpace refinement - architecture-specific` for all arches. That would be consistent with the CSpace files.